media-video/obs-studio/metadata.xml

```diff
@@ -23,6 +23,8 @@
     <flag name="ssl">Build with TLS/SSL support for RTMPS connections.</flag>
     <flag name="vlc">Build with VLC media source support.</flag>
     <flag name="websocket">Build with WebSocket API support.</flag>
+    <flag name="streamfx">Build with StreamFx plugin support (Experimental).</flag>
+    <flag name="amf">Build with AMD AMF patch (Experimental).</flag>
```

Comment:
These could be listed in an alphabetical order.

Reply:
Ok I will make the change